validator/manifest.json

```diff
@@ -15003,7 +14980,7 @@
                       "code": "Patient(http://hl7.org/fhir/StructureDefinition/patient-translated-codes).contact.relationship"
                     }
                   ],
-                  "text": "Terminology service failed while validating concept with coding(s) 'N' (system 'http://terminology.hl7.org/CodeSystem/v2-0131'): Operation validate code failed: valueset '#relationships' is unknown."
```

Comment:
Is this a reference to a contained valueset? Is our new error the one we expect here?